<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>conf</class>
 <widget class="QDialog" name="conf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CONFIGURAÇÕES</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(15, 15, 15);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>461</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_voltar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39,44,55);
color: rgb(0,0,245);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>VOLTAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_redaula">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39,44,55);
color: rgb(94,141,222);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>redefinir as aulas</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 15pt &quot;Arial Black&quot;;
color: rgb(94,141,222);</string>
   </property>
   <property name="text">
    <string>CONFIGURAÇÕES</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(15,15,15);
color: rgb(94,141,222);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Se você ver algum bug, você pode reportar ele em:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(15,15,15);
color: rgb(94,141,222);
font: 87 9pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>https://github.com/Felipe2102/Assistente-Escolar/issues/new</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(15,15,15);
color: rgb(94,141,222);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Site da escola (padrão: classrooom):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_salvar_site">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39,44,55);
color: rgb(94,141,222);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>conf</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>conf</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
